views/includes/directors.html

```diff
@@ -36,11 +36,11 @@ <h2 class="govuk-summary-card__title">{{officerCard.officer.name}}</h2>
     </dl>
       <div class="govuk-!-padding-top-3">
         {% if updateEnabled === "true" %}
-        <button id="directorUpdateButton" name="update_director_details" class="govuk-button govuk-button--secondary govuk-!-margin-right-2" data-module="govuk-button" value="{{officerCard.appointmentId}}">
+        <button id="directorUpdateButton" name="updateAppointmentId" class="govuk-button govuk-button--secondary govuk-!-margin-right-2" data-module="govuk-button" value="{{officerCard.appointmentId}}">
```

Comment:
I think there might be confusion here. I added this bit but removed it after speaking with Stokesy. The previous tickets that added the flags specified not to have this for corporate. Probably I got the information wrong but worth checking